packages/wallets/src/components/WalletListCardIActions/WalletListCardIActions.tsx

```diff
@@ -40,14 +40,39 @@ const getWalletHeaderButtons = (is_demo: boolean, handleAction?: () => void) =>
 };
 
 type TProps = {
-    account: NonNullable<ReturnType<typeof useWalletAccountsList>['data']>[number];
+    account?: NonNullable<ReturnType<typeof useWalletAccountsList>['data']>[number];
+    is_desktop_wallet?: boolean;
 };
 
-const WalletListCardIActions: React.FC<TProps> = ({ account }) => {
-    const is_demo = account.is_virtual;
+const WalletListCardIActions: React.FC<TProps> = ({ account, is_desktop_wallet = true }) => {
```

Comment:
```suggestion
const WalletListCardIActions: React.FC<TProps> = ({ account }) => {
```

We can use is_desktop / is_mobile from useDevice now right 😄